ServiceWebsite/ServiceWebsite/ClientApp/src/styles.css

```diff
@@ -81,4 +81,14 @@
 }
 .mb-0{
     margin-bottom: 0px;
-}
\ No newline at end of file
+}
+
+.vh-border {
+    padding-left: 20px;
+    margin-left: 17px;
+    border-left: 5px solid #cccccc;
+  }
+  .vh-left{
+    margin-left:13px;
+  }
+  
```

Comment:
I think that now as the same css used in two components, we shell to put it in the common style.css, maybe rename more specific.

Reply:
have added to styles.css.